domain-cc/cc-app/src/python_src/api.py

```diff
@@ -92,13 +120,15 @@ def get_classification(claim: Claim) -> Optional[PredictedClassification]:
     )
     classification_code = None
     if claim.claim_type == "claim_for_increase":
-        log_as_json({"diagnostic code": sanitize_log(claim.diagnostic_code)})
         classification_code = dc_lookup_table.get(claim.diagnostic_code, None)
 
     if claim.contention_text and not classification_code:
         classification_code = dropdown_lookup_table.get(claim.contention_text, None)
 
+    if claim.claim_type == "new":
         log_lookup_table_match(classification_code, claim.contention_text)
+    else:
+        log_as_json({"diagnostic code": sanitize_log(claim.diagnostic_code)})
```

Comment:
Should this be `diagnostic_code`?

Reply:
yeah, I think we wanted to use diagnostic code as signal for CfI and dropdown match for new, but ended up logging above message for both, so now shifting to single log message per claim